compiler-plugin/src/main/kotlin/com/google/devtools/ksp/KotlinSymbolProcessingPlugin.kt

```diff
@@ -66,8 +66,8 @@ class KotlinSymbolProcessingCommandLineProcessor : CommandLineProcessor {
             val (k, v) = value.split('=', ignoreCase = false, limit = 2)
             processingOptions.put(k, v)
         }
-        KspCliOption.KNOWN_MODIFIED_OPTION -> knownModified.addAll(value.split('=').map { File(it) } )
-        KspCliOption.KNOWN_REMOVED_OPTION -> knownRemoved.addAll(value.split('=').map { File(it) } )
+        KspCliOption.KNOWN_MODIFIED_OPTION -> knownModified.addAll(value.split(':').map { File(it) } )
```

Comment:
is this the path separator? if so, i think we should be using: https://docs.oracle.com/javase/8/docs/api/java/io/File.html#pathSeparatorChar

Reply:
I think it is the separator used [here](https://github.com/google/ksp/blob/master/gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspSubplugin.kt#L318-L319)

Reply:
Done.